pyigm/clustering/survey_2d2pcf.py

```diff
@@ -245,20 +376,23 @@ def set_normalization(self, norm, Ngal_rand=None, Nabs_rand=None):
         if norm:
             if self.gal_anly:
                 self.nDgDg = self.Ngal * (self.Ngal - 1) / 2.
-                self.nDgRg = self.Ngal * self.Ngal * Ngal_rand
-                self.nRgRg = self.Ngal * Ngal_rand * (self.Ngal * Ngal_rand - 1) / 2.
+                #self.nDgRg = self.Ngal * self.Ngal * self.Ngal_rand
+                self.nDgRg = self.Ngal * self.Ngal_rand
+                #self.nRgRg = self.Ngal * self.Ngal_rand * (self.Ngal * self.Ngal_rand - 1) / 2.
+                self.nRgRg = self.Ngal_rand * (self.Ngal_rand - 1) / 2.
 
             if self.abs_anly:
                 self.nDaDa = self.Nabs * (self.Nabs - 1) / 2.
-                self.nDaRa = self.Nabs * self.Nabs * Nabs_rand
-                self.nRaRa = self.Nabs * Nabs_rand * (self.Nabs * Nabs_rand - 1) / 2.
+                self.nDaRa = self.Nabs * self.Nabs_rand
+                self.nRaRa = self.Nabs_rand * (self.Nabs_rand - 1) / 2.
 
             if self.gal_anly & self.abs_anly:
-                self.nRaDg = self.Nabs * self.Ngal * Nabs_rand
-                self.nRaRg = self.Nabs * self.Ngal * Nabs_rand * Ngal_rand
+                self.nRaDg = self.Nabs_rand * self.Ngal
+                self.nRaRg = self.Nabs_rand * self.Ngal_rand # * Nabs_rand * Ngal_rand
                 self.nDaDg = self.Nabs * self.Ngal
-                self.nDaRg = self.Nabs * self.Ngal * Ngal_rand
+                self.nDaRg = self.Nabs * self.Ngal_rand
         else:
+            warnings.warn("This is only recommended if you are not done building up the object")
             self.nDgDg = None
             self.nDaDa = None
             self.nDaDg = None
```

Comment:
In line 275/409 (below) "abs_analy" should be "abs_anly" in the string for consistency.

Reply:
I fear I don't follow this comment.
Please elaborate.

Reply:
Its just to fix the text message of the representation in a line that is below but could not be directly commented.